Makefile

```
@@ -36,7 +36,7 @@ clean:  ## Clean build artifacts
 archive: clean  ## Create the archive for AWS lambda
 	docker build -t bucket-antivirus-function:latest .
 	mkdir -p ./build/
-	docker run -v $(current_dir)/build:/opt/mount --rm --entrypoint cp bucket-antivirus-function:latest /opt/app/build/lambda.zip /opt/mount/lambda.zip
+	docker run -v $(current_dir)/build:/opt/mount --rm --entrypoint cp bucket-antivirus-function:latest /app/build/lambda.zip /opt/mount/lambda.zip
```

Couldn't we use the variable that defines `/app`?

```suggestion
	docker run -v $(current_dir)/build:/opt/mount --rm --entrypoint cp bucket-antivirus-function:latest $(container_dir)/build/lambda.zip /opt/mount/lambda.zip
```